modules/ingestor/src/graph/sbom/mod.rs

```diff
@@ -572,7 +572,7 @@ impl SbomContext {
         let left_node_id = left_node_id.unwrap_or_else(|| self.sbom.node_id.clone());
         let right_node_id = right_node_id.unwrap_or_else(|| self.sbom.node_id.clone());
 
-        // FIXME: this could also be SPDX
+        // FIXME: this could also be CDX
```

Comment:
That comment is actually resolved. As it passes in the `ER` type now. Which handles this.